api/lib/application/sessions/session-controller.js

```diff
@@ -403,6 +403,15 @@ const deleteJuryComment = async function (request, h) {
   return h.response().code(204);
 };
 
+const dismissLiveAlert = async function (request, h) {
+  const { id: sessionId } = request.params;
```

Comment:
Proposition : renommer dans la route le param en `sessionId` et pouvoir faire ici

```suggestion
  const { sessionId } = request.params;
```